### PR TITLE
Fix acme rg as per fixes in azure-enterprise

### PR DESCRIPTION
**Pull request by adusumillipraveen**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ variables:
       value: 'azure-control-stg-rg'
     ${{ if eq( parameters['env'], 'ptlsbox') }}:
       value: 'azure-control-sbox-rg'
-    ${{ if notIn( parameters['env'], 'aat', 'perftest', 'ptlsbox', 'preview') }}:
+    ${{ if notIn( parameters['env'], 'aat', 'perftest', 'preview') }}:
```
Comment (timja) [inline azure-pipelines.yml:95]: don't you need to remove line 93 and 94 too?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -92,7 +92,7 @@ variables:
       value: 'azure-control-stg-rg'
     ${{ if eq( parameters['env'], 'ptlsbox') }}:
```
Comment (thomast1906) [inline azure-pipelines.yml:93]: Does it not still use this?

File: azure-pipelines.yml (hunk 1 above)
Comment (thomast1906) [reply azure-pipelines.yml:93]: :D literally commented same

Comment (adusumillipraveen) [reply azure-pipelines.yml:93]: Removed now :)

**Review by timja — APPROVED** [review]

**Pull request merged**